src/module/item/consumable/spell-consumables.ts

```diff
@@ -31,26 +41,36 @@ const wandCompendiumIds: Record<number, string | undefined> = {
     9: "Fgv722039TVM5JTc",
 };
 
-function getIdForSpellConsumable(type: "scroll" | "wand", heightenedLevel: number): string | null {
-    return type === "scroll"
-        ? scrollCompendiumIds[heightenedLevel] ?? null
-        : wandCompendiumIds[heightenedLevel] ?? null;
+function getIdForSpellConsumable(type: SpellConsumableItemType, heightenedLevel: number): string | null {
+    let id = null;
+    switch (type) {
+        case "cantripDeck5":
+            id = cantripDeckId;
+            break;
+        case "scroll":
+            id = scrollCompendiumIds[heightenedLevel] ?? null;
+            break;
+        default:
+            id = wandCompendiumIds[heightenedLevel] ?? null;
+    }
+    return id;
```

Comment:
return directly from inside the case statements rather than assigning to a let. You could also alternatively chain a ternary.

In general try to avoid lets whenever possible.